EIPS/eip-x.md

```diff
@@ -41,9 +41,10 @@ The existing definitions from [ERC-4626](./eip-4626.mn) apply. In addition, this
 EIP-X vaults MUST implement one or both of asynchronous deposit and redemption request flows. If either flow is not implemented in a request pattern, it MUST use the ERC-4626 standard synchronous interaction pattern. 
 
 All EIP-X asynchronous tokenized vaults MUST implement ERC-4626, with the following overrides for request flows:
-1. In asynchronous deposit Vaults, the `deposit` and `mint` methods do not transfer  `asset` to the vault, because this already happened on `requestDeposit`.
-2. In asynchronous redemption Vaults, the `redeem` and `withdraw` methods do not transfer `shares` to the vault, because this already happened on `requestRedeem`. 
-3. In asynchronous redemption Vaults, the `owner` field of `redeem` and `withdraw` MUST be `msg.sender` to prevent the theft of requested redemptions by a nonowner.
+1. In asynchronous deposit vaults, the `deposit` and `mint` methods do not transfer  `asset` to the vault, because this already happened on `requestDeposit`.
+2. In asynchronous redemption vaults, the `redeem` and `withdraw` methods do not transfer `shares` to the vault, because this already happened on `requestRedeem`. 
+3. In asynchronous redemption vaults, the `owner` field of `redeem` and `withdraw` MUST be `msg.sender` to prevent the theft of requested redemptions by a nonowner.
+4. In asynchronous vaults, it must be possible to `cancelDepositRequest` and `cancelRedeemRequest`, which reduces the `pendingDepositRequest` or `pendingRedeemRequest` to 0 and increases the `maxDeposit` and `maxRedeem` to the previous pendingRequest, allowing the user to withdraw tokens or vault shares from the initial request.
```

Comment:
I don't think this is needed. The above list are deviations where an async vault does not follow the 4626 spec. The cancelation methods instead are just extensions.
```suggestion
```

Reply:
I agree with this, this is not the place to define or clarify behavior around cancellation. Also, cancel functions are optional no?

Reply:
yeah, will move this behavior down to the cancellation request section